### PR TITLE
dev-python/toposort: 1.7 version bump

### PR DESCRIPTION
**Pull request by band-a-prend**
The `setup.py` is removed by upstream in favor of `pyproject.toml`.
Tests are now run explicitly.

### CODE REVIEW

File: dev-python/toposort/toposort-1.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+DESCRIPTION="Implements a topological sort algorithm"
+HOMEPAGE="https://gitlab.com/ericvsmith/toposort"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
```
Comment (mgorny) [inline dev-python/toposort/toposort-1.7.ebuild:19]:
```suggestion
```

File: dev-python/toposort/toposort-1.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+RESTRICT="!test? ( test )"
+
+python_test() {
+	${EPYTHON} test/test_toposort.py || die
```
Comment (mgorny) [inline dev-python/toposort/toposort-1.7.ebuild:23]:
```suggestion
	"${EPYTHON}" test/test_toposort.py || die
```

Comment (band-a-prend) [reply dev-python/toposort/toposort-1.7.ebuild]: done

File: dev-python/toposort/toposort-1.7.ebuild (hunk 1 above)
Comment (band-a-prend) [reply dev-python/toposort/toposort-1.7.ebuild]: done

**Pull request closed without merging**